doc/rst/technotes/gpu.rst

```diff
@@ -115,25 +117,35 @@ Requirements
   ``CHPL_LLVM`` must be set to ``system`` or ``bundled``). For more information
   about these settings see :ref:`Optional Settings <readme-chplenv>`.
 
-  * If using a ``system`` LLVM it must have been built with support for the
-    relevant target of GPU you wish to generate code for (e.g.  NVPTX to target
-    NVIDIA GPUs and AMDGPU to target AMD GPUs).
+  * If using a ``system`` LLVM:
+
+    * it must have been built with support for the relevant target of GPU you
+      wish to generate code for (e.g.  NVPTX to target NVIDIA GPUs and AMDGPU to
+      target AMD GPUs).
+
+    * we expect it to be the same version as the bundled version (currently 15).
+      Older versions may work; however, we only make efforts to test GPU support
+      with this version.
+
+* Either the CUDA toolkit (for NVIDIA), or ROCm (for AMD) must be installed
+  (unless using `CPU-as-Device mode`_)
 
-  * If using a system install of ``LLVM`` we expect this to be the same
-    version as the bundled version (currently 15). Older versions may
-    work; however, we only make efforts to test GPU support with this version.
+  * For targeting NVIDIA GPUs, we support CUDA versions from 10.x to 12.0
+    (inclusive).
 
-* Unless using `CPU as Device mode`_, either the CUDA toolkit (for NVIDIA), or
-  ROCm (for AMD) must be installed.
+    * If using version 10.x you must set ``CHPL_RT_NUM_THREADS_PER_LOCALE=1``.
 
-  * If targeting NVIDIA GPUs, we require CUDA toolkit to be version 10.x or 11.x
-    (inclusive). If using version 10.x you must set
-    ``CHPL_RT_NUM_THREADS_PER_LOCALE=1``. Versions as early as 7.x may work,
-    although we have not tested this.
+    * If using version 12.x you must use the bundled LLVM
```

Comment:
I was about to ask if `chpl_gpu.py` verifies this and it looks like it does:
https://github.com/chapel-lang/chapel/blob/624e5fabc5394fea6fe3f90a319d5e71ea45a929/util/chplenv/chpl_gpu.py#L247

So cool.